src/CalendarAvailability.vue

```diff
@@ -316,5 +315,8 @@ export default {
 	max-width: 500px;
 	width: 100%;
 	gap: 24px;
+	border-top: 1px solid var(--color-primary-element-light);
```

Comment:
To use the correct variable

```suggestion
	border-top: 1px solid var(--color-border);
```